lib/osf-components/addon/components/editable-field/license-manager/template.hbs

```diff
@@ -17,4 +17,5 @@
     selectedLicense=this.selectedLicense
     licensesAcceptable=this.licensesAcceptable
     isSaving=this.save.isRunning
+    onInput=(action this.noop)
```

Comment:
This `onInput` only make sense with the new manager, right? It shouldn't be here.